roxie/ccd/ccdprotocol.cpp

```diff
@@ -2148,6 +2150,13 @@ class RoxieSocketWorker : public ProtocolQueryWorker
                         response.startDataset("Tracing", NULL, (unsigned) -1);
                         msgctx->outputLogXML(response);
                     }
+                    if (statsWuid.length())
+                    {
+                        FlushingStringBuffer response(client, (protocolFlags & HPCC_PROTOCOL_BLOCKED), mlResponseFmt, (protocolFlags & HPCC_PROTOCOL_NATIVE_RAW), false, logctx);
+                        response.startDataset("Statistics", NULL, (unsigned) -1);
+                        VStringBuffer xml(" <wuid>%s</wuid>\n", statsWuid.str());
+                        response.flushXML(xml, true);
```

Comment:
The other place that calls startDataset() calls flush(), and I think that will behave differently - appending tail.  Is this correct?

Reply:
I'm not sure I understand the question, but I have checked that the answer is what I expect it to be...

<Dataset name='Result 1'>
 <Row><Result_1>1</Result_1></Row>
</Dataset>
<Statistics>
 <wuid>W20210802-102613</wuid>
</Statistics>

Reply:
I can't work out where the \</Statistics> is added from the code.  Ah!  from the destructor of the flushing string buffer.